ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/privacy/PrivateTransactionReceiptResult.java

```diff
@@ -30,6 +32,13 @@
   "from",
   "to",
   "output",
+  "publicHash",
```

Comment:
In the JI, we are calling this `commitmentHash`. Did we decide to change or was this a mistake?